overlay.py

```diff
@@ -24,6 +24,15 @@ def __init__(self, bike, width=1280, height=740):
 		self.data_update_interval = 1
 
 		self.backend = None
+		self.bg_path = None
+		if bg is not None:
```

Comment:
Maybe put this if statement after the raspberry pi check so that we can't have backgrounds on RPI? Does opencv work on raspberry pi atm?

Reply:
OpenCV works fine on raspberry pi! We use it to draw the overlay contents regardless of the backend, so it's actually already used.

Reply:
I could also see having this feature on a Pi being a useful feature if we wanted to test the overlay system on a Pi, but didn't have access to a camera module

Reply:
All good!